packages/core/less/autocomplete.less

```diff
@@ -0,0 +1,246 @@
+    position: absolute;
+    right: 0;
+    text-shadow: none;
+    top: -2px;
+  }
+}
+.rbt-token-disabled {
+  background-color: @color-pf-black-200;
+  color: @color-pf-black-500;
+  pointer-events: none;
+}
+.rbt-token-removeable {
+  cursor: pointer;
+  padding-right: 21px;
+}
+.rbt-token-active {
+  background-color: @color-pf-blue-400;
+  color: @color-pf-black-100;
+  outline: none;
+  text-decoration: none;
+}
+.rbt-loader {
+  -webkit-animation: loader-animation 600ms infinite linear;
+  -o-animation: loader-animation 600ms infinite linear;
+  animation: loader-animation 600ms infinite linear;
+  border: 1px solid #d5d5d5;
+  border-radius: 50%;
+  border-top-color: @color-pf-blue-400;
+  display: block;
+  height: 16px;
+  width: 16px;
+}
+.rbt-loader-lg {
+  height: 20px;
+  width: 20px;
+}
+.rbt-aux {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  align-items: center;
+  bottom: 0;
+  justify-content: center;
+  pointer-events: none;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 34px;
+  .rbt-close {
+    margin-top: -4px;
+    pointer-events: auto;
+  }
+}
+.rbt-aux-lg {
+  width: 46px;
+}
+.has-aux {
+  .rbt-input {
+    padding-right: 34px;
+  }
+}
+.rbt-highlight-text {
+  background-color: inherit;
+  color: inherit;
+  font-weight: bold;
+  padding: 0;
+}
+.input-group {
+  > .rbt {
+    -webkit-box-flex: 1;
+    -moz-box-flex: 1;
+    -webkit-flex: 1;
+    -ms-flex: 1;
+    flex: 1;
+    .rbt-input-hint-container {
+      display: -webkit-box;
+      display: -moz-box;
+      display: -ms-flexbox;
+      display: -webkit-flex;
+      display: flex;
+    }
+    .rbt-input-hint {
+      z-index: 5;
+    }
+    &:not(:first-child) {
+      .form-control {
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+      }
+    }
+    &:not(:last-child) {
+      .form-control {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+      }
+    }
+  }
+}
+.has-error {
+  .rbt-input-multi.focus {
+    border-color: @color-pf-red-300;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #ce8483;
+    -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #ce8483;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #ce8483;
+  }
+}
+.has-warning {
+  .rbt-input-multi.focus {
+    border-color: @color-pf-gold-600;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #c0a16b;
+    -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #c0a16b;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #c0a16b;
+  }
+}
+.has-success {
+  .rbt-input-multi.focus {
+    border-color: @color-light-green-600;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
+    -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
+  }
+}
```

Comment:
We should import the css from react-bootstrap-typeahead, then we need only include what we want to override here.